astropy/utils/iers/iers.py

```diff
@@ -92,6 +92,7 @@ def download_file(*args, **kwargs):
     """
     kwargs.setdefault('http_headers', {'User-Agent': 'astropy/iers',
                                        'Accept': '*/*'})
+    kwarg.setdefault('ftp_tls', True)
```

Comment:
Typo: should be `kwargs`

Reply:
Wait a minute, how did the test even pass with this typo?

Reply:
The tests did fail! (At least `remote_data`)  That's how I found out.

Reply:
@mhvk , sorry for the confusion, but I meant how @eteq got it to work locally.

Reply:
ah, yes, now you mention it, I wondered about that too!

Reply:
Good question.  I think it's because I tested an earlier version to prove it works and was in a hurry at the end so didn't run the later versions with ``--remote-data...`` 🐑